stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/ChannelsView.kt

```diff
@@ -0,0 +1,18 @@
+package io.getstream.chat.android.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.appcompat.widget.AppCompatTextView
+import io.getstream.chat.android.client.models.Channel
+import io.getstream.chat.android.client.models.name
+
+public class ChannelsView @JvmOverloads constructor(
```

Comment:
We use this in a lot of places, but it's [not entirely good practice](https://zsmb.co/jvmoverloads-for-android-views/)